test/components/address_component_test.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'view_component/test_case'
+
+class AddressComponentTest < ViewComponent::TestCase
+  setup do
+    VCR.use_cassette('import_test_calendar') do
```

Comment:
I think let's not touch VCR for the unit tests, anything with VCR feels more like an integration test.

Reply:
Resolved in [9b10173](https://github.com/geeksforsocialchange/PlaceCal/pull/2638/commits/9b1017332dcc523fbb76326bca1dec6ad9ba321f)